infrastructure/cli/commands/helm/history.ts

```diff
@@ -20,26 +20,40 @@ type Release = {
 }
 
 export async function handler({environment}: EnvironmentOptions): Promise<void> {
+  const rows = (await history(environment))
+    .map(({revision, updated, status, appVersion, description}) => {
+      const updatedDate = shortDateTime(updated)
+      return {revision: String(revision), updated: updatedDate, status, appVersion, description}
+    })
+  printTable(rows, [
+    {name: 'Updated', key: 'updated'},
+    {name: 'Revision', key: 'revision'},
+    {name: 'Status', key: 'status'},
+    {name: 'Version', key: 'appVersion'},
+    {name: 'Description', key: 'description'},
+  ])
+}
+
+type History = {
+  revision: number
+  updated: Date
+  status: string
+  appVersion: string
+  description: string
+}[]
+
+export async function history(environment: Environment): Promise<History> {
   const ns = namespace(environment)
   const args = ['history', releaseName, '--namespace', ns, '--output', 'json']
   const list = await helm(args, {output: 'object'}) as Release[]
-  const rows = list
+  return list
     .sort(({revision: revision1}, {revision: revision2}) => {
```

Comment:
Maybe it's just me being a bit slow as usual but I wonder if this destructuring/renaming here is a bit terse and something more explicit could be more readable, e.g.:

```javascript
.sort((release1, release2) => release2.revision - release1.revision)
// ...
```

I mean it's terse maybe but those renames in destructuring could be sneaky.

(again, just a loud thought, no reason to necessarily change, just a potential idea in case you like this more)

Reply:
ok, i'm alright with that